### ISSUE TITLE
Bump version to 0.11 for GNU ELPA

### ISSUE THREAD

**Issue by skangas** [abo-abo/ace-window#233]
It would be nice if we could get the recent changes on GNU ELPA. Maybe it is time for a bump to 0.11?

**Comment by mattiasb** [comment]
Yes please! Would love to use the the `ace-window-posframe` stuff for example!
